### ISSUE TITLE
Please see. Am I doing something wrong or do I need additions.

### ISSUE THREAD

**Issue by MaksTrader** [dynamikaweb/yii2-lightgallery#1]
I have described my problem and its solution in detail in this thread. 
https://github.com/sachinchoolur/lightGallery/issues/1168#issuecomment-927646733  

Please take a look at it. What do you think about it?       

   ],
            'options' => ['class' => 'map-conteiner'],
            'itemsOptions' => [],
            'plugins' => ['lgZoom', 'lgThumbnail', 'lgShare', 'lgFullscreen'],
            'pluginOptions' => [
                'mode' => 'lg-zoom-in-big',
                'download' => true,
                'zoom' => true,
                'closeOnTap' => true,
                'share' => true,
                'thumbnail' => true,
                'mousewheel' => true,
                'fullscreen' => true,

In short, I could not make this block work through your plugin

                mobileSettings' => [
                    'controls' => true,
                    'showCloseIcon' => true,
                    'download' => true
                ]

            ]
        ]);

Thank you very much for the plugin itself.
